src/server/pps/pretty/pretty.go

```diff
@@ -311,6 +316,8 @@ func datumState(datumState ppsclient.DatumState) string {
 		return color.New(color.FgYellow).SprintFunc()("skipped")
 	case ppsclient.DatumState_FAILED:
 		return color.New(color.FgRed).SprintFunc()("failed")
+	case ppsclient.DatumState_RECOVERED:
+		return color.New(color.FgRed).SprintFunc()("recovered")
```

Comment:
I feel like it'd make more sense for recovered to be Yellow, since it's generally less worthy of attention than a failed datum, and it's pretty equivalent in terms of importance to a skipped datum (maybe a little more worthy of attention.)

Reply:
Yeah, I would have put orange here if I could... Yellow makes sense.